src/homeblks/volume/tests/vol_gtest.cpp

```diff
@@ -1471,8 +1471,13 @@ class VolCreateDeleteJob : public TestJob {
         static thread_local std::random_device rd{};
         static thread_local std::default_random_engine engine{rd()};
         static thread_local std::uniform_int_distribution< uint64_t > dist{0, RAND_MAX};
-        if (tcfg.create_del_with_io && m_voltest->create_volume(dist(engine) % tcfg.max_vols)) { ++m_op_cnt; }
-        if (m_voltest->delete_volume(dist(engine) % tcfg.max_vols)) { ++m_op_cnt; }
+        // m_voltest->create_volume(0);
```

Comment:
Is this temporary testing code or this is intentional removing of randomization of create/del the vols?

Reply:
Yes temporary for testing. Will remove it.